src/test/java/core/basesyntax/reportcreator/CsvReporterTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.reportcreator;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvReporterTest {
+    private static final CsvReporter csvReporter = new CsvReporter();
+    private static final List<String> VALID_DATA_LIST = new ArrayList<>(2);
```

Comment:
```suggestion
    private static final List<String> validData = new ArrayList<>(2);
```
since list is not deeply immutable we should use here regular naming